sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -342,7 +342,6 @@ EventProcessorClient eventProcessorClient = new EventProcessorClientBuilder()
     .processEvent(eventContext -> {
         System.out.println("Partition id = " + eventContext.getPartitionContext().getPartitionId() + " and "
             + "sequence number of event = " + eventContext.getEventData().getSequenceNumber());
-        eventContext.updateCheckpoint(); // Update checkpoint for this partition.
```

Comment:
Can I know why we remove `updateCheckpoint()`?

Reply:
That was an error from my side. It should be fixed now.

Reply:
That was an error from my side. Sorry about it, it has been fixed now.

Reply:
got it.